proto/sf/firehose/v1/firehose.proto

```diff
@@ -0,0 +1,111 @@
+syntax = "proto3";
```

Comment:
This file should not be committed here, it gives the impression it's the canonical location for it which it not the case.

I get the point is to make the testing easier, but this should be removed.